components/inputs/input-number.js

```diff
@@ -113,8 +115,11 @@ class InputNumber extends SkeletonMixin(FormElementMixin(LocalizeCoreElement(Lit
 				?required="${this.required}"
 				?skeleton="${this.skeleton}"
 				title="${ifDefined(this.title)}"
-				.value="${this._formattedValue}"
-			><slot slot="after" name="after"></slot></d2l-input-text>
+				.value="${this._formattedValue}">
+					<slot slot="left" name="left"></slot>
+					<slot slot="right" name="right"></slot>
+					<slot slot="after" name="after"></slot>
```

Comment:
Be a little careful of spacing here... I seem to recall having to remove all the whitespace because when slots were missing there would be an extra `&nbsp;` in between them.